affinity/cpp-20/dXXX1r0.md

```diff
@@ -54,6 +54,8 @@
 
 # Abstract
 
+This paper is the result of a request from SG1 at the 2018 San Diego meeting to split P0796: Supporting Heterogeneous & Distributed Computing Through Affinity [[35]][p0796] into two separate papers, one for the high-level interface and one for the low-level interface. This paper focusses on the high-level interface; a series of properties for querying affinity relationships and requesting affinity on work being executed. [[36]][pXXXX] focusses on the low-level interface; a mechanism for discovering the topology and affinity properties of a given system.
```

Comment:
Use colon instead of semicolon for "focusses on the ... interface: a ...".  (In general, use a colon when the right-hand side of the colon defines the immediate left-hand side.)

Reply:
Thanks, fixed.